subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -477,13 +505,29 @@ static int publish_get_payload(struct mqtt_client *client, size_t length)
 static void nct_mqtt_evt_handler(struct mqtt_client *const mqtt_client,
 				 const struct mqtt_evt *_mqtt_evt)
 {
+	int err;
 	struct nct_evt evt = {
 		.status = _mqtt_evt->result
 	};
 	struct nct_cc_data cc;
 	struct nct_dc_data dc;
 	bool event_notify = false;
 
+#if defined(CONFIG_AWS_FOTA)
+	err = aws_fota_mqtt_evt_handler(mqtt_client, _mqtt_evt);
+	if (err > 0) {
+		/* Event handled by FOTA library so we can skip it */
+		return;
+	} else if (err < 0) {
+		LOG_ERR("aws_fota_mqtt_evt_handler: Failed! %d", err);
+		LOG_DBG("Disconnecting MQTT client...");
+		err = mqtt_disconnect(mqtt_client);
```

Comment:
Can have a blank line after the debug messages

Reply:
Added blank line